packages/manager/modules/bm-server-components/src/general-information/installation/gabarit/server-installation-gabarit.controller.js

```diff
@@ -134,6 +135,24 @@ export default class ServerInstallationGabaritCtrl {
     });
   }
 
+  static setSizeModalDialog(bigSize) {
```

Comment:
I'm wondering whether we should put this function in a class or anything else so that we can reuse it in several controllers without duplicating the code

Reply:
I am not able to find a proper way of doing this as creating a new component to handle modal doesn't seem pertinent, as we already have one in UI-kit but unfortunately it does not allow changing its size and I'm not sure it can be used the way your modal is used (the modal's being declared in an html which is not the one linked to the controller who's managing it)
There's also a modal in the shell, but I think using it would result in a modal covering all the page and not just the angularjs application.

Reply:
ok let's keep that